invenio_records_lom/resources/serializers/schemas.py

```diff
@@ -303,16 +303,16 @@ def get_identifiers(self, obj: LOMRecord):
 
     def get_creators(self, obj: LOMRecord):
         """Get list of creator-dicts."""
-        contributes = obj["metadata"].get("lifeCycle", {}).get("contribute", [])
+        contributes = obj["metadata"].get("lifecycle", {}).get("contribute", [])
         entities = []
         for contribute in contributes:
-            entities.append(contribute.get("entity"))
+            entities.extend(contribute.get("entity", []))
         return [{"name": entity} for entity in entities]
 
     def get_titles(self, obj: LOMRecord):
         """Get list of title-dicts."""
-        title = obj["metadata"].get("general", {}).get("title", "")
-        if not title:
+        title = obj["metadata"].get("general", {}).get("title", None)
+        if title is None:
```

Comment:
question: why the change from `if not title` to `if title is None`?

Reply:
For the first line, `metadata.general.title` should actually be a langstring-dict, the original version of defaulting to the empty string `""` made me think it's a plain `str`, which confused me for a bit.

For the second line, falsy values that aren't `None` shouldn't really be able to happen here, so I don't think there's a good reason for using any one over the other here...
`title is None`-comparison is just personal preference here.